src/main/scala/ch/openolitor/core/filestore/FileStore.scala

```diff
@@ -165,6 +165,7 @@ trait FileStore {
 class S3FileStore(override val mandant: String, mandantConfiguration: MandantConfiguration, actorSystem: ActorSystem) extends FileStore with FileStoreBucketLifeCycleConfiguration with LazyLogging {
   val opts = new ClientConfiguration
   opts.setSignerOverride("S3SignerType")
+  opts.setMaxConnections(1000)
```

Comment:
One thought about this problem: Could it be, that we have a parallel map execution? Is the order of the pages in resulting PDF always the same?

Reply:
it is parallel. It is not always respecting the same order. Do you think this is a requirement??

Reply:
I just pushed a solution to get the merged pdf ordered